recipe/meta.yaml

```diff
@@ -17,36 +17,48 @@ build:
 requirements:
   host:
     - python >=3.10
+    - pip
+    # From build-system.requires:
     - wheel
     - setuptools >=64
     - setuptools-scm >=8
-    - pip
   run:
-    - typing-extensions
+    # Main dependencies:
     - python >=3.10
     - coloredlogs
     - jax >=0.4.13
     - jaxlib >=0.4.13
     - jaxlie >=1.3.0
     - jax-dataclasses >=1.4.0
+    - pptree
+    - rod >=0.3.0
+    - typing-extensions
+    # Extra [viz] dependencies:
     - lxml
     - mediapy
     - mujoco >=3.0.0
-    - pptree
-    - rod >=0.3.0
 
 test:
   imports:
     - jaxsim
   commands:
     - pip check
     - test "$(pip list | grep jaxsim | awk '{print $2}')" == "$PKG_VERSION"
+    - pytest -vv tests/test_api_model.py
+  source_files:
+    - tests/*
```

Comment:
I don't think this is necessary, but I'll check
```suggestion
```

Reply:
The `tests` folder is not part of the `jaxsim` package, and AFAIK the original repository is not included in the conda environment created for running recipe's tests. You can check the failing CI before the commit that added these lines.

Reply:
Okay, I see. Free to merge, thanks